Core/CMakeLists.txt

```diff
@@ -96,6 +96,7 @@ set(SRCS
   Util/PPGeDraw.cpp
   CPU.cpp
   CoreTiming.cpp
+  Compatibility.cpp
```

Comment:
I think you meant to add this to /CMakeLists.txt.  Can we delete Core/CMakeLists.txt?

-[Unknown]